misc/extension_api_validation/4.1-stable_4.2-stable.expected

```diff
@@ -91,7 +91,7 @@ Added optional argument. Compatibility methods registered.
 
 GH-80954
 --------
-Validate extension JSON: Error: Field 'classes/Font/methods/find_variation/arguments': size changed value in new API, from 4 to 8.
+Validate extension JSON: Error: Field 'classes/Font/methods/find_variation/arguments': size changed value in new API, from 4 to 6.
```

Comment:
Should not be updated here, afaik, it is for 4.2 specially

Reply:
I do not think there's any way to avoid it, since validator is always comparing it to the latest version, not a compatibility method.

In this case, argument count changed in the following way: `4` (in 4.1) ⇾ `8` (in 4.2) ⇾ `6` (in this PR).

Reply:
But there's a compatibility version registered, did this error out when not adding this change?

Reply:
Yes, without this change it's giving an error in `Compatibility to 4.1-stable is broken in the following ways:` part of the script. There are compatibility methods for both `4` and `8` argument versions.

Reply:
CC @dsnopek any idea here?